workspace/apps/pidp/src/app/features/organization-info/pages/endorsements/components/endorsement-card/endorsement-card.component.scss

```
@@ -60,6 +60,9 @@ p {
         &.notice-yellow {
           background-color: pidp.$yellow;
         }
+        &.notice-red {
+          background-color: #ee494fd6;
```

As far as I know, we are supposed to follow the [BC Gov style guide](https://github.com/bcgov/design-system/blob/main/styles/colours/colourpalette.md). Use the colours from the pallet defined in `_pidp-variables.scss`.
```suggestion
        color: pidp.$red;
```